maps/adhomai/code/items/oldtimey.dm

```diff
@@ -130,3 +130,18 @@
 
 /obj/structure/window/classic/update_nearby_icons()
 	return
+
+/obj/machinery/media/jukebox/record
```

Comment:
Why is this here?

Reply:
This was here originally, actually, but maybe it got moved somehow?

Reply:
remove it then.